var/spack/repos/builtin/packages/py-cleo/package.py

```diff
@@ -12,17 +12,24 @@ class PyCleo(PythonPackage):
     homepage = "https://github.com/sdispater/cleo"
     pypi = "cleo/cleo-0.8.1.tar.gz"
 
-    version("1.0.0a5", sha256="097c9d0e0332fd53cc89fc11eb0a6ba0309e6a3933c08f7b38558555486925d3")
+    maintainers("LydDeb")
+
     version(
-        "0.8.1",
-        sha256="3d0e22d30117851b45970b6c14aca4ab0b18b1b53c8af57bed13208147e4069f",
+        "2.0.1",
+        sha256="eb4b2e1f3063c11085cebe489a6e9124163c226575a3c3be69b2e51af4a15ec5",
         preferred=True,
     )
+    version("2.0.0", sha256="fbc5cb141cbc31ea8ffd3d5cd67d3b183fa38aa5098fd37e39e9a953a232fda9")
+    version("1.0.0a5", sha256="097c9d0e0332fd53cc89fc11eb0a6ba0309e6a3933c08f7b38558555486925d3")
+    version("0.8.1", sha256="3d0e22d30117851b45970b6c14aca4ab0b18b1b53c8af57bed13208147e4069f")
 
     depends_on("python@2.7,3.4:3", type=("build", "run"))
     depends_on("python@3.7:3", when="@1:", type=("build", "run"))
     depends_on("py-poetry-core@1:", type="build")
     depends_on("py-poetry-core@1", when="@1:", type="build")
+    depends_on("py-poetry-core@1.1:1", when="@1:2.0.0", type="build")
+    depends_on("py-poetry-core@1.1.0:", when="@2.0.1", type="build")
     depends_on("py-clikit@0.6.0:0.6", when="@0.8.1", type=("build", "run"))
     depends_on("py-pylev@1.3:1", when="@1:", type=("build", "run"))
```

Comment:
Is this dep still needed?

Reply:
In pyproject.toml of py-cleo-0.8.1:
```
[build-system]
requires = ["poetry-core>=1.0.0a5"]
```

In pyproject.toml of py-cleo-1.0.0 and py-cleo-2.0.0:
```
[build-system]
requires = ["poetry-core~=1.1.0"]
```
Which exclude poetry-core@2

In pyproject.toml of py-cleo-2.0.1:
```
[build-system]
requires = ["poetry-core>=1.1.0"]
```

Reply:
My comment was on pylev, not poetry-core

Reply:
My bad, indeed it is a dependence of version 1.0.0a5.